mesh/mesh.go

```diff
@@ -344,14 +344,14 @@ func (m *Mesh) AccumulateRewards(rewardLayer LayerID, params RewardConfig) {
 
 	//todo: check if block producer was eligible?
 	for _, bl := range l.blocks {
-		if _, found := ids[bl.MinerID]; found {
+		if _, found := ids[bl.MinerID.Key]; found {
```

Comment:
It's not relevant for your PR, but we need to add a todo here that will count number of blocks by miner in case a miner is eligible for more than one in a layer